### ISSUE TITLE
Fig 1 - not accessible

### ISSUE THREAD

**Issue by matthewjay15** [UCL-ECHILD/howto.echild.ac.uk-website#25]
Should be 2 figs with text on the page so can be read by a screen reader.

**Comment by matthewjay15** [comment]
fig 1, guide 3

**Comment by matthewjay15** [comment]
Fixed
